indexer/height/client.go

```diff
@@ -1,3 +1,15 @@
 package height
 
+import (
+	tmjson "github.com/tendermint/tendermint/libs/json"
+	tmdb "github.com/tendermint/tm-db"
+)
 
+func GetLastKownHeight(indexerDB tmdb.DB) (height HeightRecord, err error) {
```

Comment:
Kown -> typo 
is this method called somewhere?

Reply:
no, it isn't.
it's a remaining from older implementation... i`ll remove it.

Reply:
@jeffwoooo removed!